plenary/plenary-26_2.yaml

```diff
@@ -0,0 +1,54 @@
+metadata:
+  title: Resolutions adopted at the meeting of ISO/TC 154 in Cologne
+  date: 2018-01-01
+  source: ISO/TC 154 Secretariat
+resolutions:
+  - dates: 
+      - 2018-01-01
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    identifier: 2017-26
+
+    actions:
+      - type: requests
+        date_effective: 2018-01-01
+        message: Appoint Mr. Ray Denneberg as WG5 Convener and project leader of ISO 8601, and request ISO/CS to formally register in ISO global data directory according to the result of voting via CIB in N901, and the email from ISO/CS, and the annotation under item 8.2 in N929
```

Comment:
I believe this plenary contains just one resolution. But I'm not sure if I have set it properly.

Source:
![capture16](https://user-images.githubusercontent.com/33627611/71760136-f3476e80-2e8e-11ea-80b3-a08e983ea8ad.JPG)

Reply:
I think there are two actions here:

```yaml
    actions:
      - type: [appoints, requests]
        date_effective: 2018-01-01
        message: Appoint Mr. Ray Denneberg as WG5 Convener and project leader of ISO 8601, and request ISO/CS to formally register in ISO global data directory according to the result of voting via CIB in N901, and the email from ISO/CS, and the annotation under item 8.2 in N929

```

Reply:
Actually we should do this:
```
    actions:
      - type: appoints
        date_effective: 2018-01-01
        message: Appoint Mr. Ray Denneberg as WG5 Convener and project leader of ISO 8601
      - type: requests
        date_effective: 2018-01-01
 request ISO/CS to formally register in ISO global data directory according to the result of voting via CIB in N901, and the email from ISO/CS, and the annotation under item 8.2 in N929
```

Reply:
Now settling to this:

```yaml
    actions:
      - type: appoints
        date_effective: 2018-01-01
        message: Appoint Mr. Ray Denneberg as WG5 Convener

      - type: appoints
        date_effective: 2018-01-01
        message: Appoint Mr. Ray Denneberg as project leader of ISO 8601

      - type: requests
        date_effective: 2018-01-01
        message: request ISO/CS to formally register in ISO global data directory according to the result of voting via CIB in N901, and the email from ISO/CS, and the annotation under item 8.2 in N929

    message: Appoint Mr. Ray Denneberg as WG5 Convener and project leader of ISO 8601, and request ISO/CS to formally register in ISO global data directory according to the result of voting via CIB in N901, and the email from ISO/CS, and the annotation under item 8.2 in N929
```